pkgs/development/tools/misc/vtable-dumper/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchFromGitHub, libelf }:
+
+stdenv.mkDerivation rec {
+  name = "vtable-dumper-${version}";
+  version = "1.2";
+
+  src = fetchFromGitHub {
+    owner = "lvc";
+    repo = "vtable-dumper";
+    rev = version;
+    sha256 = "0sl7lnjr2l4c2f7qaazvpwpzsp4gckkvccfam88wcq9f7j9xxbyp";
+  };
+
+  buildInputs = [ libelf ];
+
+  buildPhase = ''
+    make prefix=$out all
+  '';
+
+  installPhase = ''
+    make prefix=$out install
+  '';
+
```

Comment:
Could you add `makeFlags = [ "prefix=$(out)" ];` and delete the phases? (`all` is the default target of vtable-dumper.)

Reply:
Updated in all 3 packages (along with the meta improvement).  Looks a lot nicer now and I learned a new trick with the `makeFlags`; thanks @orivej!

`abi-dumper` still needs a `preBuild = "mkdir -p $out";`, since the `Makefile` shells out to `perl` to do its install and will fail if the install dir does not exist, but for the other ones I could delete everything and make it much cleaner.